pydalboard/modules/drive/distortion.py

```diff
@@ -0,0 +1,31 @@
+from typing import ClassVar
```

Comment:
This import is unused. I assume you copied it from other modules which I coded earlier, and forgot to remove the `ClassVar` import.

Reply:
Removed from the 3 drive modules.